### ISSUE TITLE
How can I install Virtual Browser in MacOS?

### ISSUE THREAD

**Issue by Dickson-Sys** [Virtual-Browser/VirtualBrowser#43]
I'm having problem to install Virtual Browser in MacOS, anyone experienced before? Or anyone have any solutions? Please give me some. Thanks

**Comment by yunxilyf** [comment]
Did you encounter any error? Please take a screenshot

**Comment by Dickson-Sys** [comment]
> Did you encounter any error? Please take a screenshot

Hello, there is no any DMG file for me to download virtualbrowser, it just showed download as EXE file

**Comment by yunxilyf** [comment]
Yes, it is not currently supported on Mac.

**Comment by Dickson-Sys** [comment]
> Yes, it is not currently supported on Mac.

Yes, that's why im having problem on this. Im using Mac right now. If not have to get Windows to download virtual browser

**Comment by yunxilyf** [comment]
mac ready
